images/redis/README.md

```diff
@@ -0,0 +1,36 @@
+# Redis
+
+[Redis](https://github.com/redis/redis) Redis is an in-memory database that persists on disk. The data model is key-value, but many different kind of values are supported: Strings, Lists, Sets, Sorted Sets, Hashes, Streams, HyperLogLogs, Bitmaps.
+
+## Get It!
+
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/redis
+```
+
+## Using Redis
+
+The default redis port is 6379.
+To run with Docker using default configuration:
+
+```sh
+docker run -p 6379:6370--rm cgr.dev/chainguard/redis
+1:C 27 Dec 2022 16:42:20.647 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
+1:C 27 Dec 2022 16:42:20.647 # Redis version=7.0.7, bits=64, commit=00000000, modified=0, pid=1, just started
+1:C 27 Dec 2022 16:42:20.647 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
+1:M 27 Dec 2022 16:42:20.648 * monotonic clock: POSIX clock_gettime
+1:M 27 Dec 2022 16:42:20.648 * Running mode=standalone, port=6379.
+1:M 27 Dec 2022 16:42:20.648 # Server initialized
+1:M 27 Dec 2022 16:42:20.650 * Ready to accept connections
+```
+
```

Comment:
Can we add a second example with a volume for the datadir, where we first run a chmod or chown on the host directory?

Reply:
Sorry, do you have an example of what you mean? The volume directive in apko does a good job here so even if you mount that `/data` directory to a hostpath it "just works".

I added an example in here!

Reply:
Yeah, if I do something like:

```
$ mkdir mydata
$ docker run -d -v $PWD/mydata:/data cgr.dev/chainguard/redis
```

Does it work? Or do I need to do a chmod/chown on `mydata`? Not sure if you need to add data and force a save to test this.

Reply:
Yea that one "just works"! Covered in an example.